src/families/hedera/test-dataset.ts

```diff
@@ -0,0 +1,92 @@
+import type { DatasetTest } from "../../types";
+import type { Transaction } from "./types";
+import {
+  NotEnoughBalance,
+  InvalidAddressBecauseDestinationIsAlsoSource,
+  AmountRequired,
+} from "@ledgerhq/errors";
+import { fromTransactionRaw } from "./transaction";
+
+const dataset: DatasetTest<Transaction> = {
```

Comment:
We will probably need to find a frozen seed

Reply:
Good point, we'll ask Hedera about it.